src/components/layout/nav.js

```diff
@@ -23,9 +23,8 @@ import {
     affiliate_signin_url,
     affiliate_signup_url,
     deriv_app_url,
+    deriv_developer_url,
     isLoggedIn,
-    getLanguage,
-    getDerivAppLocalizedURL,
 } from 'common/utility'
```

Comment:
@arome-binary reverted changes for this hasn't reflected yet

Reply:
Issue corrected!